src/components/CreateGroup/CreateGroup.connector.js

```diff
@@ -1,20 +1,22 @@
 import { connect } from 'react-redux'
 import { goBack, push } from 'connected-react-router'
 import { get } from 'lodash/fp'
+import { RESP_ADMINISTRATION } from 'store/constants'
 import { GROUP_ACCESSIBILITY } from 'store/models/Group'
 import getGroupForCurrentRoute from 'store/selectors/getGroupForCurrentRoute'
 import getMe from 'store/selectors/getMe'
 import getQuerystringParam from 'store/selectors/getQuerystringParam'
+import hasResponsibilityForGroup from 'store/selectors/hasResponsibilityForGroup'
 import { groupUrl } from 'util/navigation'
 import { createGroup, fetchGroupExists } from './CreateGroup.store'
 
 export function mapStateToProps (state, props) {
   const currentUser = getMe(state)
   const currentGroup = getGroupForCurrentRoute(state, props)
 
-  // Current user can add as parents groups that they are a moderator of or that are Open access
+  // Current user can add as parents groups ones that they are a Coordinator of or that are Open access
   const parentGroupOptions = (currentUser && currentUser.memberships.toModelArray()
-    .filter(m => m.hasModeratorRole || m.group.accessibility === GROUP_ACCESSIBILITY.Open)
+    .filter(m => m.group.accessibility === GROUP_ACCESSIBILITY.Open || hasResponsibilityForGroup(state, { groupId: m.groupId, RESP_ADMINISTRATION }))
```

Comment:
I suspect this line doesn't work? RESP_ADMIN needs to be keyed as `responsibility`. And seeing some cases where responsibility is a string and others where its an array of strings, assuming thats ok

Reply:
good catch